src/plugins/intel_gpu/include/intel_gpu/graph/serialization/layout_serializer.hpp

```diff
@@ -74,4 +74,31 @@ class Serializer<BufferType, cldnn::layout, typename std::enable_if<std::is_base
     }
 };
 
+template <typename T = std::string>
+static InferenceEngine::Layout layout_from_string(const T& name) {
```

Comment:
do we need template here?

Reply:
Actually, it was an workaround for a build error. Now I moved this function into a new file, and it's not a template any more. Thank you!